filters/reorient.cpp

```diff
@@ -112,7 +112,6 @@ tesseract_version_before_(const char *cutoff)
         {
           log::debug ("found tesseract-%1%") % result;
 
-          return (0 > strverscmp (result, cutoff));
           return (false);
```

Comment:
Does the FreeBSD C library have a string based version comparison function?  If so, that should be used here.  Or some code that does the version comparison.  Simply assuming that whatever is found is recent enough is no good.  Those tests were put in there for a reason, namely that versions earlier then the `cutoff` did not work as expected.

On your FreeBSD 12-CURRENT, the utility or library may be recent enough but what about FreeBSD 11?  Other BSDs?  Let's try to keep things portable, even for folks who use older distribution versions.

Reply:
A quick search on `strverscmp bsd` showed that the GNU Gnulib project may have a usable [substitute](https://www.gnu.org/software/gnulib/manual/html_node/strverscmp.html).